### ISSUE TITLE
Tooltips not showing

### ISSUE THREAD

**Issue by saptarshiguha** [vegawidget/vegawidget#66]
Hello,
Thank you for this very nice package. I tried the example

```
library(vegawidget)
spec_category <-
  list(
    `$schema` = vega_schema(),
    hconcat = list(
      list(data = list(values = data_category),
           mark = "bar",
           encoding = list(
             x = list(field = "category", type = "nominal"),
             y = list(field = "number", type = "quantitative")
           )
           ),
      list(data = list(values = data_category),
           mark = "bar",
           encoding = list(
             x = list(field = "category", type = "nominal"),
             y = list(field = "number", type = "quantitative")
           )
           )
    ))
as_vegaspec(spec_category)
```

it renders correctly but i dont see any hover over tool tips. If i "open in vega editor" , i do see the tooltips. I get the following error in the JS console

```
The compiled spec uses Vega v3, but current version is 4.0.0. vega-embed-modified.js:1:44008
```
The issue is present in both Firefox and Safari.
Cheers

**Comment by ijlyttle** [comment]
Hi, 

This is because vegawidget uses Vega-Lite 2.6, and tooltips are not in the default until Vega-Lite 3.0.

As you may know, they have just officially released Vega-Lite 3.0, so we will be updating this package in the near future.

Thanks!

**Comment by saptarshiguha** [comment]
Much appreciated! Thanks for the heads up
